ISSUE_TEMPLATE.md

```diff
@@ -4,5 +4,5 @@
  - Explain the issue / bug as understandably as possible.
  - If possible add minecraft, forge, java and modpack version numbers for debugging (mod authors might need these).
  - If the issue/bug is recurring explain how it can be recreated (helps darkosto!).
- - If possible add error/crash log to [Pastebin](https://pastebin.com/) and link it here (if you disslike pastebin any other similiar site is fine too).
- - If possible add a video/gif of the issue/bug (makes it easier for darkosto to understand you).
\ No newline at end of file
+ - If possible add error/crash log to [Pastebin](https://pastebin.com/) and link it here (if you dislike pastebin any other similiar site is fine too).
```

Comment:
And apparently `similar` isn't written as `similiar` ;)

Reply:
Doing good grammar is hard when your tired and a bit drunk 😂